### ISSUE TITLE
Incorrect Connections Between Uppercase Cursive "E" and Specific Lowercase Letters

### ISSUE THREAD

**Issue by toni-estevez** [RosaWagner/Borel#15]
Hello,

I have noticed an issue with the connections in the Borel typefaces that include an uppercase cursive "E", specifically when certain lowercase letters follow this letter. The exit stroke of the uppercase cursive "E" does not align correctly with the entry stroke of the following lowercase letters: b, f, h, i, j, k, l, p, r, s, t, x, and z.

Thank you for your attention and for your work on this beautiful font that I use in the preparation of my students' activity sheets.

Best regards.

Editing: The same issue occurs with the uppercase cursive letters C, T, and X.

**Comment by RosaWagner** [comment]
There is a special feature for these connections, did you activate the contextual alternates? If yes, then I may have made a mistake before exporting

edit: I checked the last proof and the connections are good so it should work if you activate the contextual alternates. Please send a screenshot if you still see a problem after that

**Comment by kenmcd** [comment]
A guess...
In Microsoft Word Contextual Alternates are not On by default.
So you need to enable them in the text style.
Modify Style > Format > select Font > Advanced tab > check Use Contextual Alternates.

![Word-Enable-Contextual-Alternates](https://github.com/user-attachments/assets/a416343e-1d26-4522-a6db-cb1a4eba321b)

Then the Borel connections should work as expected.

Per OpenType specifications Contextual Alternates is supposed to be On by default, so Word is a bit odd here by setting it to Off.

**Comment by toni-estevez** [comment]
I would like to apologize for the confusion caused by my earlier report. After rechecking the issue, I realized that the problem was not with the Borel font itself. The incorrect connections I observed were due to formatting interference in a .docx document I opened with LibreOffice in Linux, specifically caused by the "horizontal kerning" font feature disabled by default.

When testing the font in a clean .odt document, I found no issues with the connections. Everything works perfectly.

Thank you for your understanding, and I appreciate your work on this excellent typeface!
